### PR TITLE
add trivy_exposedsecrets_info metric with discovered secrets details

### LINKED ISSUE #998: Additional labels for trivy_image_exposedsecrets metric
Hello folks 👋🏼 

I was looking at working on a PR to add additional labels to the `trivy_image_exposedsecrets` metrics, specifically we would like to be able to identify better the secrets that were found and use it in our alerting and dashboards.

I wanted to reach to see if this is something you'd be interested in?

I'd like to add these additional labels:
- secret_target
- secret_rule_id
- secret_category
- secret_title
(perhaps also secret_match in case the above may produce non unique results?)

I'm also debating if these should be added to the existing metric, or perhaps add a separate (optional?) metric like in the case of `trivy_vulnerability_id`? Personally, I don't think this is needed but would appreciate some guidance here.

Thanks!

/cc @chen-keinan

### PR DESCRIPTION
**Pull request by dkulchinsky**
## Description

Adding a new metric: `trivy_exposedsecrets_info` that provides details about discovered secrets

The metric exposes the following labels (in addition to the base set):
- secret_category
- secret_rule_id
- secret_target
- secret_title

## Related issues
- Close #998

Remove this section if you don't have related PRs.

## Checklist
- [x] I've read the [guidelines for contributing](https://github.com/aquasecurity/trivy-operator/blob/main/CONTRIBUTING.md) to this repository.
- [x] I've added tests that prove my fix is effective or that my feature works.
- [x] I've updated the [documentation](https://github.com/aquasecurity/trivy-operator/tree/main/docs) with the relevant information (if needed).
- [x] I've added usage information (if the PR introduces new options)
- [ ] I've included a "before" and "after" example to the description (if the PR is a user interface change).

### CODE REVIEW

**Comment by dkulchinsky** [comment]
note I didn't change anything in `Chart.yaml` which I assume would be taken care of when a new release is prepped, but the chart was updated to support the new configuration flag and env variable.

**Comment by dkulchinsky** [comment]
Hey @chen-keinan 👋 this is ready for review 🙏🏼

**Comment by chen-keinan** [comment]
@dkulchinsky thanks , will review it later on

File: pkg/operator/etc/config.go [hunk 1]
```diff
@@ -25,6 +25,7 @@ type Config struct {
 	MetricsBindAddress                           string         `env:"OPERATOR_METRICS_BIND_ADDRESS" envDefault:":8080"`
 	MetricsFindingsEnabled                       bool           `env:"OPERATOR_METRICS_FINDINGS_ENABLED" envDefault:"true"`
 	MetricsVulnerabilityId                       bool           `env:"OPERATOR_METRICS_VULN_ID_ENABLED" envDefault:"true"`
+	MetricsExposedSecretInfo                     bool           `env:"OPERATOR_METRICS_EXPOSED_SECRET_INFO_ENABLED" envDefault:"true"`
```
Comment (chen-keinan) [inline pkg/operator/etc/config.go:28]: `MetricsExposedSecretInfo` should be false (optional) by default right ?

**Comment by chen-keinan** [comment]
@dkulchinsky  pr is looking good , I have added one comment

File: pkg/operator/etc/config.go (hunk 1 above)
Comment (dkulchinsky) [reply pkg/operator/etc/config.go:28]: ahh, good point 👍🏼 

I copied over `MetricsVulnerabilityId` which looks like is set to default true as well, should I change it to false as well?

**Review by chen-keinan — APPROVED** [review]

**Comment by chen-keinan** [comment]
lgtm 🚀

**Pull request merged**